### DISCUSSION TITLE
nedelec元

### DISCUSSION THREAD

**Discussion by guihunkun** [weihuayi/fealpy#48] [Feature request]
请问魏老师是否已经实现nedelec元？有demo程序吗？

**Comment by weihuayi** [comment]
有 nedelec 元的程序了， 但是程序应该还有问题，只有最低次验证过，高次的还没有测试通过。
可以看 test/FirstKindNedelecFiniteElementSpace2dTest.py

**Comment by guihunkun** [comment]
好的，谢谢老师，我看一下，这个验证解的是二维的拉普拉斯方程？

**Comment by scaomath** [comment]
> 好的，谢谢老师，我看一下，这个验证解的是二维的拉普拉斯方程？

curl rot u - u = f, and `u` is a vector function.
